org/eclipse/persistence/eclipselink/maven-metadata.xml

```diff
@@ -8,7 +8,8 @@
     <versions>
       <version>2.7.4.payara-p2</version>
       <version>2.6.5.payara-p4</version>
+      <version>2.7.6.payara-p1</version>
```

Comment:
I might be wrong - but I can't locate artifacts (pom, jar etc) of that version in https://github.com/payara/Payara_PatchedProjects/tree/master/org/eclipse/persistence/eclipselink. Is that on purpose?

Reply:
I don't believe you're wrong - something seems to have gone wrong with the mvn deploy in that it's only updated the metadata files.
@Cousjava can you look into this please?